lib/api/channel.js

```diff
@@ -359,6 +359,59 @@ class Channel extends ApiResource {
       `/channels/${channelId}/featured-videos.json`,
       qs.stringify({video_ids: videoIds}))
   }
+
+  /**
+   * Getting the channel's recording status
+   * While broadcasting we can have the video it's recorded to
```

Comment:
> "Getting the channel's recording status"

Please simplify: "Get the channel's recording status"

> "While broadcasting we can have the video it's recorded to"

This description is hard to understand. Please give more detail.